source/manual/alerts/rabbitmq-no-consumers-listening.html.md

```diff
@@ -4,75 +4,89 @@ title: 'RabbitMQ: No consumers listening to queue'
+Check the logs for the `rabbitmq` application which can be achieved logging
+into Kibana and searching for `application: rabbitmq`. Is there evidence of any
+errors?
 
-If consumers are idle, messages sent to the queue will not be being processed.
-This means that the actions which are meant to happen in response to the
-messages will not happen. For example, email alerts about updated content will
-not be sent if the `email_alert_sevice` queue isn't running.
+### Check the RabbitmQ Grafana dashboard
 
-Unless there is a wider RabbitMQ failure, messages will not be lost - they will
-be processed once the problem is resolved.
+This [Grafana dashboard][rabbitmq_grafana_dashboard] shows activity across
+multiple exchanges and queues. The main exchange we expect to be monitoring is
+`published_documents` which handles broadcasting to services such as search and
+email-alert-service when content changes across GOV.UK.
 
-## Investigation
+Looking at the queue graphs we should look out for the following:
 
-RabbitMQ has an admin interface, which allows details of recent activity on the
-queues to be seen. To log in, [follow the instructions in the RabbitMQ
-docs][rabbitmq_control_panel].
+* **Check for high 'ready' messages for the queue** - indicates these messages are
+  waiting to be processed in RabbitMQ by the consumer (e.g
+  email-alert-service).
 
-The admin interface allows you to see details (and graphs!) of the messages
-sent to each queue, and the number of messages held on the queues. Look at the
-queue for which the alert happened. If no messages have been sent to the queue
-over the last several minutes, the most likely failure is that the heartbeat
-messages are no longer being sent correctly. Look for recent changes to the
-[Publishing API][publishing_api].
+* **Check for high 'unacknowledged' messages for the queue** - implies that
+  messages have been read by the consumer but the consumer has never sent back
+  an ACK to the RabbitMQ broker to say that it has finished processing.
 
-If messages have been received, and the queue has messages held on it and not
-being processed, the consumers have either got stuck or stopped running for
-some reason. Restarting the consumers for the relevant app will probably clear
-this up (but please make sure the team owning the app are also alerted, since
-this shouldn't normally happen). For example:
+* **Check for high 'redeliver' rate for the queue** - in the event of network
+  failure (or a node failure), messages can be redelivered. An example is if
+  the consumer dies (its channel is closed, connection is closed, or TCP
+  connection is lost).
 
-```sh
+If we're seeing high 'redeliver' rates, high 'ready' or 'unacknowledged'
+messages then this could indicate an issue with the consumer.
+
+### Troubleshooting steps
+
+1. You could try restarting the consumer applications. After restarting, check
+   to see if the problem is solved, if not, check to see if any errors are
```

Comment:
```suggestion
```

(we say this in the next step, although it can be irrespective of whether the service is running)

Reply:
Updated the text on both steps! 👍